includes/wc-core-functions.php

```diff
@@ -1995,7 +1997,8 @@ function wc_make_phone_clickable( $phone ) {
  * @return mixed Value sanitized by wc_clean.
  */
 function wc_get_post_data_by_key( $key, $default = '' ) {
-	return wc_clean( wp_unslash( wc_get_var( $_POST[ $key ], $default ) ) ); // @codingStandardsIgnoreLine
+	// phpcs:ignore
```

Comment:
In this case I was just using `ignore` because there are four errors in the same line, but ok, I'll include all of them.

Reply:
Or even better, since three of those are in the `WordPress.Security.ValidatedSanitizedInput` category and it's very clear that we don't care about input sanitization for that line, I'll just disable the category itself.